src/components/Navigator/NavigatorResource.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from "react";
+import { fromVtt } from "subtitles-parser-vtt";
```

Comment:
Does this package have any @types we can import via [Definitely Typed](https://www.typescriptlang.org/dt/search?search=)?

Reply:
And after checking out this package `https://www.npmjs.com/package/subtitles-parser-vtt`, curious if there are any more mature options to pull data out of our WebVTTs?   I'm fine either way, but a package with roughly 100 weekly downloads with a version of `0.1.0` could conceivably go away, not get updated, more bug-prone, etc.   Your call, just thinking out loud.